### DISCUSSION TITLE
How to use command : test_token ?

### DISCUSSION THREAD

**Discussion by joroMaser** [jonassjoh/dspW245#10] [Q&A]
How to use command : test_token ?

**Comment by jonassjoh** [comment]
I don't know. The list of commands I've posted in some places I've just yanked from the source code. There is a function with a bunch of if-statements, each statement checking for each command in the list. Since I never had any use for most of the commands in my own project, I never bothered to investigate them. My memory is not as fresh as it was, but I don't think that all commands were implemented, even thought they were present in the code. You'd have to do some trial and error, or dive into the source code yourself.

What is it that you are trying to do?

**Comment by joroMaser** [comment]
Ok , and can you please explain that does the command `send_register` do ?

**Reply by jonassjoh** [reply]
I don't know. Unless the command is present in the `dspW245.py` script, I would only be guessing as to how you would use it. If you're lucky @Garfonso might know, I know he's implemented some extra commands over in his repository. You can find a link to it at the bottom of the README.

**Reply by Garfonso** [reply]
> Ok , and can you please explain that does the command `send_register` do ?

I don't have a clue... for most methods I did not figure out what parameters the expect so they only returned errors to me..
